buildenv/jenkins/common/build.groovy

```diff
@@ -379,6 +380,17 @@ def archive_sdk() {
                                 "target": "${ARTIFACTORY_CONFIG['uploadDir']}",
                                 "props": "build.buildIdentifier=${BUILD_IDENTIFIER}"]
                 specs.add(debugImageSpec)
+                // Extract obfuscation change log files to Artifactory
+                def findZelixOut = sh(script: "find ${makeDir} -name 'ObfuscateLog*.txt' | sed 's#.*/##'", returnStdout: true).trim()
+                def zelixFiles = findZelixOut.tokenize('\n');
+                if (!zelixFiles.isEmpty()) {
+                    for (String zelixFile in zelixFiles) {
+                        def zelixSpec = ["pattern": "${makeDir}/${zelixFile}",
+                                         "target": "${ARTIFACTORY_CONFIG['uploadDir']}",
+                                         "props": "build.buildIdentifier=${BUILD_IDENTIFIER}"]
```

Comment:
This won't work properly if the files are not in the `${makeDir}` directory (and instead in some subdirectory). I suggest removing the `sed` filter and use the full name as the `"pattern"` value.

Reply:
Should I assume that the files will always have the top directory of `make`?

Reply:
Each line produced by `find` should be a path starting with `${makeDir}` (i.e. `make`). You should also filter by type:
```
        def findZelixOut = sh(script: "find ${makeDir} -name 'ObfuscateLog*.txt' -type f", returnStdout: true).trim()
```

Reply:
Testing suggested changes now...